linux_os/guide/services/ssh/var_sshd_max_sessions

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
```

Comment:
The file name still needs to have the `.var` extension so that it is recognized as a Value, so it should be `var_sshd_max_sessions.var`.

Reply:
I really do appreciate that you all are not killing me as I learn.  Your guidance and assistance is much appreciated.

Resolved in 0e95def